fiaisis/frontend
# Add home page and import parent application themeing

Closes #7, closes #14, closes #25, closes #26.

This PR adds a home page from which to use Interactive Reduction. It also imports themeing from the parent application so that switching between light / dark mode as well as changing the contrast should affect certain elements on the page.

## How to test:

Pull from SciGateway's `develop` branch so it is up to date. For the plugin to use the new home page, update `settings.json` in your SciGateway repo by adding `"homepageUrl": "/ir"`.

Build the plugin then run it through SciGateway. The home page should load first which contains links to the other pages. 2 tabs should be available on the sidebar for the "Instruments", and "History" pages (the latter of which should be a blank page).

Using the settings cog wheel at the top, switching between light / dark and contrast mode should cause page elements to change to appopriate colours.

## Known bugs:

`GlobalStyles.tsx` is not properly applying SciGateway's light / dark mode styling and needs to be looked at closer.
